src/translation/cn/page.json

```diff
@@ -0,0 +1,400 @@
+{
```

Comment:
You've structured this content based on the app's folder structure. What is the justification to this and what are the alternatives?

Reply:
It's easier for me to track with this folder structure even i have some repeated content under the different categories, so that i know what are done and what are not, and how much content on each session, for now i haven't came up with a better way to structure it other than breaking the categories down to different files. But not sure what is reason to do it now